src/user_lists/views.py

```diff
@@ -1,25 +1,51 @@
 from django.db import IntegrityError
-from rest_framework import viewsets
-from rest_framework.mixins import CreateModelMixin
+from django.utils import timezone
+from rest_framework import status, viewsets
+from rest_framework.mixins import CreateModelMixin, DestroyModelMixin, UpdateModelMixin
 from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
 from rest_framework.serializers import ValidationError
 
+from researchhub.permissions import IsObjectOwner
+
 from .models import List
 from .serializers import ListSerializer
 
 
+def _update_list_timestamp(list_obj, user):
```

Comment:
We don't need `updated_by`.

Also, we don't need to update `updated_date` manually. should just happen....